core/src/main/scala/ai/h2o/sparkling/H2OContext.scala

```diff
@@ -488,6 +493,11 @@ object H2OContext extends Logging {
           s" but your Spark is of version ${sc.version}. Please make sure to use correct Sparkling Water for your" +
           s" Spark and re-run the application.")
     }
+    if (isSparkVersionDeprecated) {
+      logWarning(
+        "Apache Spark 2.1.x is deprecated and " +
```

Comment:
I would make this parametrized by the current version `SparkSessionUtils.active.version` as well

Reply:
will do